cves/kernel/CVE-2021-27363.yml

```diff
@@ -458,7 +461,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: "commons users should have access to everything"
```

Comment:
Just... What? I recommend looking over his example of this section as it goes much more in-depth than a single sentence.  It also sounds like you're implying everyone should have admin privileges, which violates least_privilege